README.md

```diff
@@ -29,7 +29,37 @@ Faster for session establishment, not throughput ! SSH3 offers a significantly f
 <i>SSH3 (top) VS SSHv2 (bottom) session establishement with a 100ms ping towards the server.</i>
 </p>
 
-## 🔒 SSH3 is secure
+## ⚡ SSH3 is an experimental prototype
+
+While SSH3 shows promise for faster session establishment, it is 
+still at an early proof-of-concept stage. As with any new complex 
+protocol, **expert cryptographic review over an extended timeframe is required before reasonable security conclusions can be made**.
+
+We are developing SSH3 as an open source project to facilitate community feedback and analysis. However, we **cannot yet endorse its appropriateness for production systems** without further peer review. Please collaborate with us if you have relevant expertise!
+
+## 🥷 Your SSH3 server should not yet be made public
```

Comment:
what about ?

> ## 🥷 Do not deploy the SSH3 server on your production servers for now

It is a suggestion, it is okay if you want to refine it, let's iterate.